.github/workflows/docker-build.yaml

```
@@ -43,11 +42,134 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
 
+      - name: Analyze disk usage and cleanup
+        run: |
+          echo "=== Initial disk usage ==="
+          df -h
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)